package.json

```diff
@@ -7,7 +7,7 @@
     "doc": "docs"
   },
   "scripts": {
-    "lint": "eslint -- --fix frontend/src/**/*js frontend/cypress/**.js && stylelint frontend/src/**/*.scss",
+    "lint": "eslint -- --fix frontend/src/**/*js frontend/cypress/**.js && stylelint frontend/src/**/*.scss && pug-lint frontend/src/*.pug && pug-lint frontend/src/tabs/*.pug",
```

Comment:
will `pug-lint frontend/src/*.pug frontend/src/tabs/*.pug` be the same?

Reply:
I tried using `pug-lint frontend/src/*.pug` but this doesn't check the ones inside the `tabs` folder. Same for vice-versa too

Reply:
As in, just use `pug-lint frontend/src/*.pug frontend/src/tabs/*.pug` instead of `pug-lint frontend/src/*.pug && pug-lint frontend/src/tabs/*.pug`

Reply:
Aah I see. Thanks for bringing it to my notice